incubator/etcd/templates/statefulset.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
       - name: {{ template "etcd.fullname" . }}
-        image: "{{ .Values.Image }}:{{ .Values.ImageTag }}"
+        image: nearform/etcd:3.2.11
```

Comment:
why are we moving away from values file here?

Reply:
Changing the image might break the chart. 

@phyrwork  commented 15 days ago
Decouples image from chart source. New image could break chart. Should remain as fixed version

Reply:
@Elexy The image should be configurable. It's part of the [review guidelines](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration). One simple use case is those who copy the image to their own repository rather than using the upstream one.